jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletHolder.java

```diff
@@ -155,6 +155,11 @@ public ServletHolder(Class<? extends Servlet> servlet)
         setHeldClass(servlet);
     }
 
+    public Object getMultipartConfig()
```

Comment:
Why returning `Object`?

Reply:
I've added the type, but it is lowercase p to match the Servlet API type now: `MultipartConfigElement`